src/clj/orcpub/email.clj

```diff
@@ -25,11 +25,14 @@
   [{:type "text/html"
     :content (hiccup/html (verification-email-html first-and-last-name username verification-url))}])
 
+(def truthy? #{"true"})
```

Comment:
If we're not going to use it anywhere else, it should probably not get a name and a var just yet. We can move the set `#{"true"}` to where `truthy?` is down below. Also, this doesn't match the current concept of truthy/falsy in Clojure, where falsy is `nil` or `false`, and truthy is anything else. It won't handle the inevitable user who uppercases it as "TRUE", either.

Alternatively, you could turn it into a fn that intelligently converts strings to booleans, and place it in a utility namespace. This is what the [cuerdas library's `to-bool`](https://funcool.github.io/cuerdas/latest/#to-bool) fn does. We could copy that or include cuerdas.